### ISSUE TITLE
Project disappears after removing all roles

### ISSUE THREAD

**Issue by jackmen** [inception-project/inception#1205]
Hi,

we installed inception on a server and assigned different roles to people.
I have manager, curator, and user right. 

1. I created a project
2. In order to see only the agreement of two different annotators and not  my own, I wanted to remove me as annotator. I unchecked without thinking all my roles and the project disappeared. 
3. There was no way for me getting access to the project from within inception. 
4. Our admin was also not able to solve this within inception
5. There was no warning that this could happen when unchecking. 
6. We solved it on the level of the mysql database and everything went back to normal. 

Please have a look at this as users might end up removing themselves from their projects and a bit of experience is necessary to get everything back in order. The admin should always be able to access created projects even if the curator has expulsed himself by mistake ;-)!

Besides this, it is a very great tool and a huge step forward!

Thanks,

**Comment by reckart** [comment]
An admin user should be able to access a list of *all* projects via the *Administration* link in the top-right part of the title bar. There the admin should be able to see all projects and also be able to modify the settings of all projects including the user permissions - even for projects in which the admin has no permissions at all.

An admin user should be unable to revoke his admin status on the user management page.

What did your admin try?

**Comment by jackmen** [comment]
Thanks for the quick feedback,

yes, with admin I mean admin of the entire installation.
Indeed we did not realize the options on the administration page. We intuitively entered the project on the left pane and there were no options. We recreated the above explained situation and solved it using the admin access via administration to restore the user in the project and it worked fine. 

But as you mentioned above, to place a warning before removing oneself form a project, would be a good add.

Thanks and have a nice WE!
